<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="searchTerm">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>classic</string>
    </property>
   </widget>
   <widget class="QLabel" name="search">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Term:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="rawText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputText">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>421</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>49</y>
      <width>771</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QPushButton" name="GetStreams">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Streams</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetSearch">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Quit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
